book/forms.rst

```diff
@@ -1568,6 +1568,15 @@ are 4 possible *parts* of a form that can be rendered:
 
 .. note::
 
+    The portion corresponding to ``collection`` type is ``entry``.
+    In the case of embedded collections, for example,
+    in a form which you have a year which contains months, and months contains days.
+    You want to personalize each collection render, where the form prefix is ``_my_form_year``,
+    you need to write two blocks, ``_my_form_year_months_entry_row`` to customize month render, and
+    ``_my_form_year_months_entry_days_entry_row`` for days inputs.
```

Comment:
@Th3Mouk Those are nice examples! However I suggest minor things if they should go in the docs:
- utiliser des noms de classes et d'options en anglais "saison => season",
- actually `PeriodeSemaineType` may break
  - This is a good opportunity to show some option resolving as default for "saison" option is null but you call `$options['saison']->getId()`
    You should have in `configureOptions`
    ```php
    $resolver->setDefaults(array(
        'data_class' => 'Entities\ElcaBundle\Entity\PeriodeSemaine',
        'saison' => 'null',
        'query_builder' => function (Options $options) { // make it lazy
            $saison = $options['saison'];
            if (null  === $saison || ! ($saison instanceof Saison) {
                return null;
            }
            
            return function (PeriodeRepository $em) use ($saison) {
                return $em
                    ->createQueryBuilder('p')
                    ->join('p.saison', 's')
                    ->where('s.id = :id_saison')
                    ->setParameter('id_saison', $saison->getId())
                ;
            };
        },
    ));

    Btw `CalendrierExerciceType` only needs to have `$resolver-setDefined('saison')` as a "child" option it needs no default value, only to be passed to the child which will resolve it.